playwright/src/main/java/com/microsoft/playwright/impl/ListenerCollection.java

```diff
@@ -41,7 +41,9 @@ <T> void notify(EventType eventType, T param) {
     }
 
     for (Consumer<?> listener: new ArrayList<>(list)) {
-      ((Consumer<T>) listener).accept(param);
+      if (listener != null) {
```

Comment:
Having `null` listeners in the list is an error, we should instead throw when the client attempts to add `null` as the listener.

Reply:
changed